WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductVariationFormViewModel.swift

```diff
@@ -26,6 +26,12 @@ final class ProductVariationFormViewModel: ProductFormViewModelProtocol {
         isUpdateEnabledSubject.eraseToAnyPublisher()
     }
 
+    /// Unused in variations but needed to satisfy protocol
+    var isEligibleForBlazeUpdate: AnyPublisher<Void, Never> {
+        isEligibleForBlazeSubject.eraseToAnyPublisher()
```

Comment:
As this is unused, we may not need to add implementation `isEligibleForBlazeSubject`. We can return a `Just(Void())` as a placeholder.

```suggestion
        Just(Void()).eraseToAnyPublisher()
```

That way, we can remove `isEligibleForBlazeSubject` below.

Reply:
I learned something new here, thanks!